lib/cli.js

```diff
@@ -543,6 +543,53 @@ yargs
             Mcdev.fixKeys(argv.BU, argv.TYPE, csvToArray(argv.KEY));
         },
     })
+    .command({
+        command: 'updateNotifications <BU> <TYPE> [KEY]',
+        aliases: ['updNotif'],
```

Comment:
how about "un" as alias?
```suggestion
        aliases: ['un'],
```